ci/build.sh

```diff
@@ -68,6 +68,10 @@ elif [ -n "$DOCKER_PASSWORD" ] && [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS
   echo "Pushing alpha docker image with version $new_alpha"
   echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
   docker push $DOCKER_TAG:$new_alpha
+
+  echo "Tagging git with v$new_alpha..."
+  git tag -a "v$new_alpha" -m "Bump version to v$new_alpha"
+  git push --tags origin master
```

Comment:
Would this create too many tags in github? Do we have any concern there?

Reply:
It will create a tag for every PR that gets merged. I think that's fine though -- we can always re-evaluate if we hit some limit.

Reply:
+1 I'm okay with this, it helps keep track of which alpha tag corresponds to which commits as well